docs/application/dotnet/tutorials/certificates/permit-device-to-install-apps.md

```diff
@@ -2,7 +2,7 @@
 
 In the menu bar, click **Tools** > **Tizen** > **Tizen Device Manager**.
 
-And right-click anywhere in the file explorer of the **Device Manager** and select **Permit to install applications**.
+Right-click anywhere in the file explorer of the **Device Manager** and select **Permit to install applications**.
 
 ![Device Manager](media/DeviceManager.png)
 
```

Comment:
Line 11:
Can we rephrase as:

If failed to install application to the device, follow the instructions:
•	Verify the device date and time. 
If the certificate issued is before the device date and time, the application cannot be installed, as the certificate is considered invalid.
•	Verify whether the permit to install applications process is performed correctly.

Please verify whether the technical meaning is altered.

Reply:
done

Reply:
Sorry, hj-na-park , I can't see your update related with this.
Would you please re-check?

Reply:
I'll apply it next patch. Please check latest version.

Reply:
line 14 : Would you please add 3 spaces in front of this sentence? This line is sub item of above line, so indentation is needed.

@Reni-90 , after @hj-na-park update this PR, please check the latest version and approve this PR if every your comments are applied.

Reply:
Q) line 14 : Would you please add 3 spaces in front of this sentence? This line is sub item of above line, so indentation is needed.
A) Done.